src/Routing/AdminCreateHtmlRouteProvider.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\Routing\AdminCreateHtmlRouteProvider.
+ */
+
+namespace Drupal\entity\Routing;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+/**
+ * Provides HTML routes for creating entities using the administrative theme.
+ *
+ * @see \Drupal\entity\Routing\CreateHtmlRouteProvider.
+ */
+class AdminCreateHtmlRouteProvider extends CreateHtmlRouteProvider {
```

Comment:
good idea, IMHO the @see is a bit pointless because its just pointing to the parent class